.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: ruby
+rvm:
+  - 2.3.0
```

Comment:
We could add a few more Ruby versions here. For instance, our core project is still running on `2.1.x`.